spec/60_show_created_by.lua

```diff
@@ -0,0 +1,29 @@
+feature("#60 Show created/updated by", function()
+    before_scenario(function()
+        when(todo, "show_button"):then_return(true)
+    end)
+
+    after_scenario(function()
+        todo.show_button:revert()
+
+        -- clear all tasks
+        global.todo.open = {}
+        global.todo.done = {}
+    end)
+
+    scenario("A task has a creator/modifier", function()
+        local player = game.players[1]
+
+        local task = todo.create_task("Test")
+        task.by = player.name
```

Comment:
here you use .by, below in the frame you use .created_by. How does this work together?

Reply:
Well this is a missed correction.

Reply:
Test updated with both fields